layouts/partials/search-input.html

```diff
@@ -1,5 +1,7 @@
-{{ if or .Site.Params.gcs_engine_id .Site.Params.algolia_docsearch -}}
+{{ if .Site.Params.gcs_engine_id -}}
 <input type="search" class="form-control td-search-input" placeholder="&#xf002; {{ T "ui_search" }}" aria-label="{{ T "ui_search" }}" autocomplete="off">
+{{ else if .Site.Params.algolia_docsearch -}}
+<div id="docsearch"/>
```

Comment:
Hmm, I don't think that a `div` can be a self-closing tag. And ... SO seems to agree: https://stackoverflow.com/q/7971716/3046255

Reply:
Done.